lib/utils/log-deprecation.js

```diff
@@ -144,7 +144,11 @@ module.exports.printSummary = async () => {
       fse.ensureDir(path.dirname(healthStatusFilename)),
       fsp.writeFile(tmpHealthStatusFilename, healthStatus.join('\n')),
     ]);
-    await safeMoveFile(tmpHealthStatusFilename, healthStatusFilename);
+    try {
+      await safeMoveFile(tmpHealthStatusFilename, healthStatusFilename);
+    } catch (e) {
```

Comment:
- Let's use full names for variables (so `error`)
- Let's log error with our logging interface (`log.error('Cannot store information on approached deprecation. Please ensure process has the write access to `~/.servelress` directory')`)
- Let's log the error only in `EACCESS` case and for all other cases let it throw

Reply:
"Let's log the error only in EACCESS case and for all other cases let it throw"
Sorry, Im not a dev, Can you help me understand what that means?

Reply:
Failure that relates to permissions is reported with `EACCES` code, and we can confirm that we deal with a such error by inspecting `error.code` (which will be equal to `"EACCES"` then)

Reply:
eh, I had a stab at this, but again, im not a dev. if Ive missed the mark, let me know where.